Firestore/Example/Tests/SpecTests/json/bundle_spec_test.json

```diff
@@ -3,7 +3,8 @@
     "describeName": "Bundles:",
     "itName": "Bundles query can be loaded and resumed from different tabs",
     "tags": [
-      "multi-client"
+      "multi-client",
+      "no-ios"
```

Comment:
It has no-ios tag, but still get parsed and extracted to ios sdk 🤔

Reply:
It will get extracted, but should not get executed by the ios spec test runner.